_posts/2019-12-20-festive-message.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title:  "A look back at 2019"
+date: 2019-12-20
+author: strimzi_developers
+---
+
+The last year has been a busy one for Strimzi with many highlights. 
+We wanted to reflect on and celebrate a few of them.
+
+<!--more-->
+
+Many features have been added during the year. Here are some of the key ones:
+
+* Storage - resizing persistent volumes and support for JBOD (including adding and removing volumes)
+* Monitoring - support for [Jaeger](https://www.jaegertracing.io/) tracing, [Prometheus](https://prometheus.io/) metrics and exposing consumer lag using [Kafka Exporter](https://github.com/danielqsj/kafka_exporter)
+* Support for OAuth2 authentication
+* HTTP access through the Kafka Bridge
+* New mechanisms for off-cluster access
+
+Next year, Strimzi will accelerate and we are looking forward to adding support for:
+
+* Cluster rebalancing using [Cruise Control](https://github.com/linkedin/cruise-control)
+* An Operator for handling Kafka Connect connectors as native Kubernetes resources
+* Enabling ZooKeeper rolling upgrade
```

Comment:
This is confusing, because we've always supported ZK rolling _restart_ (which sometimes get referred to as upgrade). Maybe better to call it "Supporting upgrade to ZooKeeper to 3.5.x"?